CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## Unreleased
+# 1.160.0 (January, 19 2024)
+ENHANCEMENTS:
+* resource/spotinst_ocean_aws: Added `parameters` object under `tasks`
```

Comment:
please modify the sentence to read as:
Added support for `parameters` object under `scheduling_tasks` block